fe/fe-core/src/main/java/org/apache/doris/catalog/HiveTable.java

```diff
@@ -74,6 +89,92 @@ public Map<String, String> getHiveProperties() {
         return hiveProperties;
     }
 
+    @Override
+    public List<Column> getBaseSchema(boolean full) {
+        if (isLocalSchema) {
+            return super.getBaseSchema(full);
+        }
+        List<Column> hiveMetastoreSchema = Lists.newArrayList();
+        try {
+            for (FieldSchema field : HiveMetaStoreClientHelper.getSchema(this)) {
+                field.getType();
+                hiveMetastoreSchema.add(new Column(field.getName(), convertToDorisType(field.getType()),
+                        true, null, true, null, field.getComment()));
+            }
+        } catch (DdlException e) {
+            LOG.warn("Failed to get schema of hive table. DB {}, Table {}. {}",
+                    this.hiveDb, this.hiveTable, e.getMessage());
+            return null;
+        }
+        fullSchema = hiveMetastoreSchema;
+        return fullSchema;
+    }
+
+    public Column getColumn(String name) {
```

Comment:
Unused function.

Reply:
This is function is used in describe table and query statement. It is override the function in Table

Reply:
OK